toucan_connectors/toucan_connector.py

```diff
@@ -26,7 +26,20 @@ class Category(str, Enum):
 
 class DataSlice(NamedTuple):
     df: pd.DataFrame  # the dataframe of the slice
-    total_count: int  # the length of the raw dataframe (without slicing)
+    total_count: Optional[
```

Comment:
yes we should delete and update the slices returned elsewhere

Reply:
:+1: